src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -339,15 +339,23 @@ public function start($max = null)
      * Advances the progress output X steps.
      *
      * @param int $step Number of steps to advance
-     *
-     * @throws LogicException
      */
     public function advance($step = 1)
     {
         $this->setProgress($this->step + $step);
     }
 
     /**
+     * Regresses the progress output X steps.
+     *
+     * @param int $step Number of steps to regress
+     */
+    public function regress($step = 1)
```

Comment:
But it would keep it consistent with the progress property too, although I'm fine with both, I don't see anything shocking about using negative here.